2996_reflection/reflection.md

```diff
@@ -4993,12 +4988,22 @@ consteval vector<info> members_of(info r);
 * a function,
 * a non-static data member,
 * a namespace, or
-* a namespace alias.
+* a namespace alias,
+
+and if its first declaration is within a definition of `$E$`.
 
 [Counterexamples of representable members include: injected class names, partial template specializations, friend declarations, and static assertions.]{.note}
 
+[#]{.pnum} A member `$M$` of a class or namespace is _members-of-visible_ from a point `$P$` if there exists a declaration `$D$` of `$M$` that is reachable from `$P$`, and either `$M$` is not TU-local or `$D$` is declared in the translation unit containing `$P$`.
+
+
 [#]{.pnum} *Effects*: If `dealias(r)` represents a class template specialization with a reachable definition, the specialization is instantiated.
 
+[#]{.pnum} *Returns*: A `vector` containing reflections of all _members-of-representable_ members of the entity represented by `r` that are _members-of-visible_ from one of the points in the evaluation context ([expr.const]).
+If `$E$` represents a class `$C$`, then the vector also contains reflections representing all unnamed bit-fields declared within the member-specification of `$C$`.
+Non-static data members are indexed in the order in which they are declared, but the order of other kinds of members is unspecified.
```

Comment:
> are indexed in the order in which they are declared

Again maybe this is normal (for core wording) but "are ordered in the order they were declared" or something of that ilk seems better (if possible).

Reply:
I'm using existing language that we've had for `members_of` and `bases_of`, but I do think I prefer the word "indexed" - it conveys a notion of assigning indices 0, 1, 2, 3, ... in the same order that the members are declared.